onepwd/src/onepwd/__init__.py

```diff
@@ -308,6 +325,16 @@ def get_single_secret(op, item_name, field=None, vault=None):
               svalue=f["v"]
     return svalue
 
+# used in the ansible action 'update_s3_secret' 
+def get_secret_values_list(op, item_name,  vault=None):
+    item=op.get('item', item_name, vault=vault)
+    svalue=""
+    if item["templateUuid"]=='005': # Password template type
+        svalue=item["details"]["sections"][0]["fields"] 
+    else:
+        print("Error - is the template a password template?")      
```

Comment:
You should throw an exception here. In the worst case you prints an error and returns an empty password value.

Reply:
done